CHANGELOG.md

```diff
@@ -1,6 +1,27 @@
 # Change Log
 This file documents all notable changes to juttle-service. The release numbering uses [semantic versioning](http://semver.org).
 
+## 0.3.0
+Release 2016-02-26
+
+### Major changes
+NOTICE: This release includes breaking changes to the wire format.
+- Update to reflect changes in Juttle 0.5.0. [[#54]](https://github.com/juttle/juttle-service/pull/54)
+- Convert all extendable-base classes to ES6 classes [[#41]](https://github.com/juttle/juttle-service/pull/41)
+- Update all remaining references to 'sink' in JSDP messages to 'view'. *This changes the JSDP wire format*. [[#37]](https://github.com/juttle/juttle-service/issues/37)
+
+### Minor changes
+- Add a `/version` endpoint that returns version information on the juttle and adapter modules in use [[#56]](https://github.com/juttle/juttle-service/pull/56)
```

Comment:
can you update this to https://github.com/juttle/juttle-service/pull/62 (or list both)?